src/Engine/GameEvents/TurnInitEvent.java

```diff
@@ -14,14 +17,21 @@
   Commander cmdr;
   int turn;
   boolean opaque;
-  String msg;
+  Collection<String> msg;
 
   public TurnInitEvent(Commander co, int turnNum, boolean hideMap)
   {
-    this(co, turnNum, hideMap, "Turn "+turnNum);
+    this(co, turnNum, hideMap, new ArrayList<String>());
+    msg.add("Turn "+turnNum);
```

Comment:
This didn't need to change, though?

Reply:
Hah, yeah. I changed this before I re-added the "string" constructor. Oh well.